i18n/en.json

```diff
@@ -824,6 +824,14 @@
     "translation": "shrug"
   },
   {
+    "id": "api.command_help.desc",
+    "translation": "Open the Mattermost help page"
```

Comment:
In JIRA ticket, /help command open the help page.
goto location redirect user to webpage